torch_geometric/nn/conv/rgcn_conv.py

```diff
@@ -59,6 +59,9 @@ class RGCNConv(MessagePassing):
         compensate.
         We advise to check out both implementations to see which one fits your
         needs.
+        RGCNConv can use dynamic shapes. Dynamic shape means that the shape of
+        the interim tensors can be determined at runtime. If your device
+        doesn't support dynamic shapes, use FastRGCNConv.
```

Comment:
Still a little confused
1. `FastRGCNConv` also support dynamic shape, right? Like you can pass in `x` of shape `[in_channels, any shape]` and internally the shape of `x_i` depends on number of edges.
2. Have you tested this on some specific device where `FastRGCNConv` works and `RGCNConv` doesn't.

Reply:
1. I dont see any usage of dynamic shape. Can you send a place in code where it is used?
2. Yes, I tested it on device without support dynamic shapes. `FastRGCNConv` works and `RGCNConv` doesnt.